# Task 4.

You are given two random variables X and Y.
E(X) = 0.5, Var(X) = 2
E(Y) = 7, Var(Y) = 3.5
cov (X, Y) = -0.8

Find the variance of the random variable Z = 2X - 3Y

In [10]:
import numpy as np

In [2]:
x = round(np.random.rand(),4)
y = round(np.random.rand(),4)

In [3]:
print(x,y)

0.7512 0.6607


In [4]:
expectation_x = 0.5
expectation_y = 7

In [5]:
var_x = 2
var_y = 3.5

In [6]:
cov_x_y = -0.8

V(aX−bY)=a2V(X)+b2V(Y)−2abCov(X,Y)

In [7]:
first_element = 2**2*var_x
second_element = 3**2*var_y
third_element = 2*2*3*cov_x_y

In [8]:
result = first_element + second_element - third_element

In [9]:
result

49.1

Our result is 49.1 according to all formulas of basic statistics.